### ISSUE TITLE
Some questions about the backbone and train settings

### ISSUE THREAD

**Issue by keSang3** [linhanwang/SCCNet#8]
Thank you for your excellent work! I have a few questions. I noticed that you mentioned in your paper that The backbone is trained together with 10 times smaller learning rate. But in the forward stage use " with torch.no_grad():" when extracting image features. so how do the backbone parameters update together during training? And why are the learning parameters of SCCNet using different backbone networks in the paper table all remain 5.2 M? When I trained with VGG16 as Backbone, I only got 27.78 and 33.47 mIoU for fold 0 1/5shot . The training settings used the same as mentioned in README. Are there any other settings that need to be changed to replicate the results of the paper?

**Comment by linhanwang** [comment]
I didn't fix the seed and best parameter for the training. You may have to tune some parameters to get better results.

Another thing is that the result in the paper is after merging with spectral segmentation.

The number of trainable parameters doesn't include the backbone.

**Comment by keSang3** [comment]
Thanks for your reply, so the training actually freeze the backbone network, right? You said that you didn't fix the seed and best parameter for the training. Does that mean the learning rate, batch size and other parameters given in the paper are not the best. I need to tune the parameters by myself.

**Comment by linhanwang** [comment]
> Thanks for your reply, so the training actually freeze the backbone network, right? You said that you didn't fix the seed and best parameter for the training. Does that mean the learning rate, batch size and other parameters given in the paper are not the best. I need to tune the parameters by myself.

From my experience, it's almost impossible to totally make the training process deterministic, even when you fix the seed. I released the checkpoint to reproduce the results. But I lost them when migrating from different cloud storages.
